python/cudf/io/json.py

```diff
@@ -0,0 +1,219 @@
+        datelike columns default is True; a column label is datelike if
+        * it ends with ``'_at'``,
+        * it ends with ``'_time'``,
+        * it begins with ``'timestamp'``,
+        * it is ``'modified'``, or
+        * it is ``'date'``
+    keep_default_dates : boolean, default True
+        If parsing dates, then parse the default datelike columns
+    numpy : boolean, default False
+        Direct decoding to numpy arrays. Supports numeric data only, but
+        non-numeric column and index labels are supported. Note also that the
+        JSON ordering MUST be the same for each term if numpy=True.
+    precise_float : boolean, default False
+        Set to enable usage of higher precision (strtod) function when
+        decoding string to double values. Default (False) is to use fast but
+        less precise builtin functionality
+    date_unit : string, default None
+        The timestamp unit to detect if converting dates. The default behaviour
+        is to try and detect the correct precision, but if this is not desired
+        then pass one of 's', 'ms', 'us' or 'ns' to force parsing only seconds,
+        milliseconds, microseconds or nanoseconds respectively.
+    encoding : str, default is 'utf-8'
+        The encoding to use to decode py3 bytes.
+    lines : boolean, default False
+        Read the file as a json object per line.
+    chunksize : integer, default None
+        Return JsonReader object for iteration.
+        See the `line-delimted json docs
+        <http://pandas.pydata.org/pandas-docs/stable/io.html#io-jsonl>`_
+        for more information on ``chunksize``.
+        This can only be passed if `lines=True`.
+        If this is None, the file will be read into memory all at once.
+    compression : {'infer', 'gzip', 'bz2', 'zip', 'xz', None}, default 'infer'
+        For on-the-fly decompression of on-disk data. If 'infer', then use
+        gzip, bz2, zip or xz if path_or_buf is a string ending in
+        '.gz', '.bz2', '.zip', or 'xz', respectively, and no decompression
+        otherwise. If using 'zip', the ZIP file must contain only one data
+        file to be read in. Set to None for no decompression.
+
+    Returns
+    -------
+    result : Series or DataFrame, depending on the value of `typ`.
+
+    See Also
+    --------
+    .to_json
+    """
+
+    warnings.warn("Using CPU via Pandas to read JSON dataset, this may "
+                  "be GPU accelerated in the future")
+
+    pd_value = pd.read_json(
+        path_or_buf=path_or_buf,
+        orient=orient,
+        typ=typ,
+        dtype=dtype,
+        convert_axes=convert_axes,
+        convert_dates=convert_dates,
+        keep_default_dates=keep_default_dates,
+        numpy=numpy,
+        precise_float=precise_float,
+        date_unit=date_unit,
+        encoding=encoding,
+        lines=lines,
+        chunksize=chunksize,
+        compression=compression
```

Comment:
If our defaults all match, it seems like `**kwargs` is indeed preferable.

Reply:
ACK, will fix it across all of them.